Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -4697,6 +4697,31 @@ static NSPasteboardName pasteboardNameForAccessCategory(WebCore::DOMPasteAccessC
     return validAttributes.get().get();
 }
 
+static Vector<WebCore::CompositionUnderline> extractUnderlines(NSAttributedString *string)
```

Comment:
This is the same as the stuff in the `#else` of  `compositionUnderlines`, right? If so, the `#else` section could probably be reduced to just a call to `extractUnderlines`

Reply:
Good idea I didn't realize that. I'll just remove `extractUnderlines` and then this PR becomes a much smaller change.